### PR TITLE
Plone 5 fixtures

### LINKED ISSUE #27: Behavior not found
After activating collective.taxonomy (1.4.2+ from master) I get the following error accessing the Behavior Tab of a content-type:

http://localhost:8080/Plone5/dexterity-types/Document/@@behaviors

```
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module plone.z3cform.layout, line 66, in __call__
  Module plone.z3cform.layout, line 50, in update
  Module z3c.form.form, line 226, in update
  Module plone.z3cform.patch, line 21, in BaseForm_update
  Module z3c.form.form, line 154, in update
  Module z3c.form.form, line 136, in updateWidgets
  Module z3c.form.field, line 277, in update
  Module z3c.form.browser.checkbox, line 45, in update
  Module z3c.form.browser.widget, line 171, in update
  Module z3c.form.widget, line 234, in update
  Module z3c.form.widget, line 106, in update
  Module z3c.form.datamanager, line 76, in query
  Module z3c.form.datamanager, line 71, in get
  Module plone.app.dexterity.browser.behaviors, line 41, in __getattr__
  Module plone.behavior.registration, line 76, in lookup_behavior_registration
BehaviorRegistrationNotFound: collective.taxonomy.testtaxonomy
```

### PR DESCRIPTION
**Pull request by tomgross**
This PR
- Fixes #27 
- Starts with current language when creating a taxonomy
- Fixes failure on creating taxonomy, if site is set to extended language (en_US)
- Switch from elemttree to lxml

### CODE REVIEW

**Comment by tomgross** [comment]
**Attention!** Since the name of the behaviour changes, this might not work with current vocabularies and might need an upgrade step.

File: src/collective/taxonomy/tests/robot/test_taxonomy.robot [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+Library  DateTime
+
+Test Setup  TestSetup
+Test Teardown  TestTeardown
+
+
+*** Test Cases ***************************************************************
+
+Scenario: As a manager I can add a taxonomy
+  Given loged in manager
```
Comment (vincentfretin) [inline src/collective/taxonomy/tests/robot/test_taxonomy.robot]: loged -> logged

File: src/collective/taxonomy/utility.py [hunk 2]
```diff
@@ -14,33 +15,34 @@
...
+from zope.interface import implementer
 from zope.lifecycleevent import modified
 
 import generated
 import logging
 
 from copy import copy
 
-from collective.taxonomy import PATH_SEPARATOR
+from os import sep as PATH_SEPARATOR
```
Comment (vincentfretin) [inline src/collective/taxonomy/utility.py]: Here PATH_SEPARATOR is used for the title of the term "level1 / level2 / level3"
Your change doesn't make sense. On windows you will have "\" as separator.
Please keep "from collective.taxonomy import PATH_SEPARATOR"
I monkey patch this constant in my project to change the separator to '>' so to have "level1 > level2 > level3". The use case is described in the README.

**Comment by vincentfretin** [comment]
I don't fully understand what you changed.
Why the behavior name changed? From what to what? Can you give an example?

**Comment by tomgross** [comment]
Thanks @vincentfretin for the review. I created another PR (#31) with a better documented and cleaner history.

**Pull request closed without merging**